ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -386,15 +386,37 @@ function upgrade() {
             echo "Current cluster is on ${cluster_src_ver}"
         fi
         echo "Negative Testing: upgrade to an unsigned image without --force option"
-        admin_ack
-        cco_annotation
+        admin_ack "${SOURCE_VERSION}" "${TARGET_VERSION}"
+        cco_annotation "${SOURCE_VERSION}" "${TARGET_VERSION}"
         run_command "oc adm upgrade --to-image=${TARGET} --allow-explicit-upgrade"
         error_check_invalid_image
         clear_upgrade
         check_upgrade_status "${cluster_src_ver}"
     fi
     run_command "oc adm upgrade --to-image=${TARGET} --allow-explicit-upgrade --force=${FORCE_UPDATE}"
     echo "Upgrading cluster to ${TARGET} gets started..."
+
+    # https://issues.redhat.com//browse/OTA-861
+    # When upgrade is processing, Upgradeable will be set to false
+    if [[ "$TARGET_MINOR_VERSION" -le 18 ]]; then
+        return 0
+    fi
+    local case_id="OCP-25473"
```

Comment:
To be consistent with other steps and not introduce unexpected break to `upgrade` func, suggest to keep the checkpoints in `check_upgrade_status`.